docs/release-notes/7.0.md

```diff
@@ -0,0 +1,228 @@
+# Release notes for Gluster 7.0
+
+This is a major release that includes a range of code improvements and stability
+fixes along with a few features as noted below.
+
+A selection of the key features and changes are documented in this page.
+A full list of bugs that have been addressed is included further below.
+
+- [Announcements](#announcements)
+- [Major changes and features](#major-changes-and-features)
+- [Major issues](#major-issues)
+- [Bugs addressed in the release](#bugs-addressed)
+
+## Announcements
+
+1. Releases that receive maintenance updates post release 7 are, 5, 6 and 7
+([reference](https://www.gluster.org/release-schedule/))
+
+2. Release 7 will receive maintenance updates around the 10th of every month
+for the first 3 months post release (i.e Sep'19, Oct'19, Nov'19). Post the
+initial 3 months, it will receive maintenance updates every 2 months till EOL.
+
+
+
+## Major changes and features
+
+### Highlights
+
+- Several stability fixes addressing,
+  - coverity, clang-scan, address sanitizer and valgrind reported issues
+  - removal of unused and hence, deprecated code and features
+- Performance Improvements
+
+
+Features 
+
+#### 1. Rpcbind not required in glusterd.service when gnfs isn't built.
```

Comment:
Reorder this.
(2) -> (1) Add text on what options are needed and what should be set to enable or use the feature
(3) -> (2) Is this by default, or are there options that enable this?
(1) -> (5) 
(4) remove issue # and if we have started an implementation, should it feature here? Are there ways in which users can test/use what is in the code base now? If so need documentation to help with the same